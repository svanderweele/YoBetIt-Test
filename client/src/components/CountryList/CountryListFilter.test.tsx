import React from "react";
import { shallow } from "enzyme";
import CountryListFilter from "./CountryListFilter";
describe("CountryList", () => {
  it("should render correctly", () => {
    const component = shallow(
      <CountryListFilter
        onFilterExactMatch={(props: any) => {}}
        onFilterNameChanged={(props: any) => {}}
      />
    );
    expect(component).toMatchSnapshot();
  });
});
