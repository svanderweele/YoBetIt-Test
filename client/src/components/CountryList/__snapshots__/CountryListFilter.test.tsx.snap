// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CountryList should render correctly 1`] = `
<InputGroup
  className="mb-2"
>
  <InputGroupPrepend>
    <InputGroupText
      id="country-name"
    >
      Country Name
    </InputGroupText>
  </InputGroupPrepend>
  <FormControl
    aria-describedby="country-name"
    aria-label="Country Name"
    onChange={[Function]}
    placeholder="Malta, Sweden"
  />
  <InputGroupAppend>
    <InputGroupText
      id="country-name"
    >
      Exact match?
    </InputGroupText>
    <InputGroupCheckbox
      aria-label="exact-match-country-name"
      onChange={[Function]}
    />
  </InputGroupAppend>
</InputGroup>
`;
